h2o-test-support/src/main/java/water/TestUtil.java

```diff
@@ -617,6 +617,12 @@ static Frame trackIfScopeActive(Frame frame) {
     return frame;
   }
 
+  public static void assertExists(String fname) {
+    NFSFileVec v = makeNfsFileVec("smalldata/airlines/AirlinesTrain.csv.zip");
```

Comment:
Shouldn't `fname` be passed to `makeNfsFileVec` instead of the constant path?

Reply:
holy moly - absolutely! thank you fro catching that